homework_6_191122/src/Circle.java

```diff
@@ -0,0 +1,19 @@
+// Napisat' class Circle s parametrami radiysa, method polycheniya i zadaniya radiysa i vuchisleniya
+// ploshadi kryga.
+
+public class Circle {
+    double radius;
+    double areaCircle;
+
+    public double setRadius(double radius) {
+        return this.radius = radius;
+    }
+
+    public double getRadius() {
+        return radius;
+    }
+
+    public double getAreaCircle() {
+        return this.areaCircle = Math.PI * Math.pow(radius, 2);
```

Comment:
don't write like this, you can either assign variable or return a value

Reply:
Fixed. Changed to **return Math.PI * Math.pow(radius, 2);**